<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>684</width>
    <height>535</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(104, 38, 38);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
    <layout class="QVBoxLayout">
      <item alignment="Qt::AlignCenter">
        <widget class="QLabel" name="label">
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 255, 255);
          font: 20pt &quot;MS Shell Dlg 2&quot;;</string>
          </property>
          <property name="text">
           <string>User</string>
          </property>
        </widget>
      </item>
      <item alignment="Qt::AlignCenter">
        <widget class="QPushButton" name="graph_bt_2">
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 255, 255);
          font: 16pt &quot;MS Shell Dlg 2&quot;;</string>
          </property>
          <property name="text">
           <string>Show Graphs</string>
          </property>
        </widget>
      </item>
      <item alignment="Qt::AlignCenter">
        <widget class="QPushButton" name="graph_bt">
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 255, 255);
          font: 16pt &quot;MS Shell Dlg 2&quot;;</string>
          </property>
          <property name="text">
           <string>Show Charts</string>
          </property>
        </widget>
      </item>
      <item alignment="Qt::AlignCenter">
        <widget class="QPushButton" name="serachFlights">
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 255, 255);
          font: 16pt &quot;MS Shell Dlg 2&quot;;</string>
          </property>
          <property name="text">
           <string>Search Flights</string>
          </property>
        </widget>
      </item>
    </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>684</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
